tests/test_load_results/test_mteb_load_results.py

```diff
@@ -114,9 +106,10 @@ def test_load_results_main_score_in_real_results(task):
     ],
 )
 def test_load_results_scores(task, dataset):
+    """Test that all keys from actual task results presented in real task result"""
```

Comment:
Hmm instead I don't think this is something we actually want to test for. It prevents us from adding new metrics. I would delete this test. The only thing we really want to test for is the main metric e.g. "accuracy" is there.